### PR TITLE
update README: reorganize and dev instructions

### PR DESCRIPTION
**Pull request by dlebauer**
* This should have all of the commands required to get BrAPI running on your the computer. 
* also reorganized some content and moved sql commands to contrib/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,24 +14,44 @@ select id, sitename from sites
 | /germplasm  | cultivars.  |       |
```
Comment (KristinaRiemer) [inline README.md:14]: Is the period at the end of cultivars a typo?

File: README.md [hunk 2]
```diff
@@ -88,24 +14,44 @@ select id, sitename from sites
...
 | /observations | traits | |
 
-## Contributed Data
+## How to set up a development environment.
 
-This repository provides the canonical reference for data that is
-outside of the scope of databases used in the TERRA REF program. Such
-data can be found in the `/contrib/` folder. 
+We assume you have the following installed:
+- Docker (to run BETYdb)
```
Comment (KristinaRiemer) [inline README.md]:
```suggestion
- [Docker](https://www.docker.com/products/docker-desktop) (to run BETYdb)
```

File: README.md [hunk 3]
```diff
@@ -88,24 +14,44 @@ select id, sitename from sites
...
+- [Postman](https://www.postman.com/) (to test out APIs)
+- [PyCharm](https://www.jetbrains.com/pycharm/) (for development)
 
-Genomics data in `contrib/genomics` is in a set of CSVs that were
-previously only available in the [experimental design section of the
-TERRA REF documentation](https://docs.terraref.org/scientific-objectives-and-experimental-design/experimental-design). These files provide metadata that describe the germplasm used in the sorghum trials, and were originally prepared by Noah Fahlgren. 
+### Run a development instance of BETYdb
+
+The following commands can be used to initialize the database to be used with BRAPI development. This will fetch public data from TERRA-REF database (id=6). To get the latest data you can call the sync command.
```
Comment (KristinaRiemer) [inline README.md:26]:
```suggestion
The following commands can be used to initialize the database to be used with BRAPI development. Run these commands from within the root directory of a this repository. These commands will fetch public data from the TERRA-REF instance of the BETYdb database (id=6). To get the latest data you can call the sync command. 
```

File: README.md [hunk 4]
```diff
@@ -88,24 +14,44 @@ select id, sitename from sites
...
-name of the endpoint, for example /locations will need a file called
-api/locations.py. In there you will need a `search` function to
-handle the call to the GET endpoint. If the function is not
-implemented the url will return the actual name of the file and the
-function name.
+Now that you have BETYdb running on port 5432, you will be able to begin developing the server. These instructions assume that you are using PyCharm and have a project open to your brapi directory.
+
+* Go to run --> edit configurations
+* Set the "name" field to "server"
+* Set the 'script path' to `<localpath>/brapi/server.py`
```
Comment (KristinaRiemer) [inline README.md:42]:
```suggestion
* Set the 'script path' to `<localpath>/brapi/server.py` (localpath is file path to local `server.py`) 
```

File: README.md [hunk 5]
```diff
@@ -88,24 +14,44 @@ select id, sitename from sites
...
+* In the top right of PyCharm, select the server configuration and click the green triangle to run the server
```
Comment (KristinaRiemer) [inline README.md:43]: I think it selects the new configuration automatically, so I might replace "select the server configuration" to "make sure server configuration `server` is selected".

File: README.md [hunk 6]
```diff
@@ -88,24 +14,44 @@ select id, sitename from sites
...
+* In Postman test that the API exists by navigating to http://localhost:5000/brapi/v1/calls  
+
```
Comment (KristinaRiemer) [inline README.md:45]:
```suggestion

When you open `requirements.txt` in PyCharm, you will be prompted to install any required Python packages that are not already installed. 

```

**Review by KristinaRiemer — APPROVED** [review]
I suggested a few small edits in the README, otherwise it was easy to follow and everything worked! (Minus dealing with Python libraries)

File: README.md (hunk 5 above)
Comment (dlebauer) [reply README.md:43]: updated -

File: README.md [hunk 7]
```diff
@@ -88,24 +14,44 @@ select id, sitename from sites
...
-api/locations.py. In there you will need a `search` function to
-handle the call to the GET endpoint. If the function is not
-implemented the url will return the actual name of the file and the
-function name.
+Now that you have BETYdb running on port 5432, you will be able to begin developing the server. These instructions assume that you are using PyCharm and have a project open to your brapi directory.
+
+* Go to run --> edit configurations
+* Set the "name" field to "server"
+* Set the 'script path' to `<localpath>/brapi/server.py`
+* In the top right of PyCharm, select the server configuration and click the green triangle to run the server
```
Comment (dlebauer) [inline README.md:43]:
```suggestion
* In the top right of PyCharm, make sure the configuration server is selected. Then click the green triangle to run the server
```

**Pull request merged**